application-xpoll-ui/src/main/resources/XPoll/XPollSheet.xml

```diff
@@ -93,9 +93,38 @@
         #if($isChecked)X#end
       #else
         &lt;label&gt;
-          &lt;input type="#if($isRadio)radio#{else}checkbox#end"
-            #if($isCurrentUser)id="xpoll-proposal$proposal.hashCode()"#end name="$user" value="$proposal" $checked
-            $disabled /&gt;
+         ## condorcet method
+         #if ($pollObj.getValue('type') == '2')
+           #set ($indexOfProposal = $votes.indexOf($proposal) + 1)
+           #set ($noOption = '')
+           #if ($indexOfProposal &lt;  1)
+             #set ($noOption = 'selected')
+           #end
+
+           #if ($isCurrentUser &amp;&amp; $pollObj.getValue('status') == 'active')
+             &lt;select class="xpollSelect" proposal="$proposal"&gt;
+               &lt;option disabled $noOption value&gt;$services.localization.render('contrib.xpoll.vote.select')&lt;/option&gt;
```

Comment:
Something looks wrong here with `value`

Reply:
I wanted to show that it had no value/had empty value.

Reply:
I'd simply remove the value. Raises less questions.